runtime/neumann/src/lib.rs

```diff
@@ -1103,23 +1103,32 @@ impl_runtime_apis! {
 					let action_enum = pallet_automation_time::Action::Notify {
 						message: "get_fees".into()
 					};
-					AutomationTime::calculate_execution_fee(&action_enum, executions)
+					match AutomationTime::calculate_execution_fee(&action_enum, executions) {
```

Comment:
charles changed calculate_execution_fee to return a [result wrapped balance](https://github.com/OAK-Foundation/OAK-blockchain/pull/188/commits/a3ff79880bdb50a4b0a655cc69331fa4232ecf55#diff-e6a13f7032a383e19b77d20043ca46041d94f5e49396812e2de1b2000c301e59R1141) because of checked_add vs the saturated_add request from the audit. so now i have to unwrap. i don't wanna use .unwrap() because if it errors, unwrap will panic and kill our RPC node, which we don't wanna happen. so i'm just returning 0 if somehow we end up with a number that overflows a u128. otherwise, it returns the correct balance. is this fine?

Reply:
also should never overflow, since execution_times is limited to u32. i took the largest possible value for u32 and input into execution_times and it seems to not overflow u128 on the result.

![image](https://user-images.githubusercontent.com/22332240/179032231-feda33c6-9124-4744-9266-b5af055a03b7.png)

if we go higher than u32, it returns rpc error:
![image](https://user-images.githubusercontent.com/22332240/179032389-5dde7c44-2905-4478-8eb4-e1ef703863d4.png)